### PR TITLE
Improve descriptions of flow parameters 

### PR DESCRIPTION
**Pull request by j616**
# Details
Improve descriptions of flow parameters 

# Pivotal Story (if relevant)
Story URL: https://www.pivotaltracker.com/story/show/185727931

# Related PRs
_Where appropriate. Indicate order to be merged._

# Submitter PR Checks
_(tick as appropriate)_

- [ ] PR completes task/fixes bug
- [ ] API version has been incremented if necessary
- [ ] ADR status has been updated, and ADR implementation has been recorded
- [ ] Documentation updated (README, etc.)
- [ ] PR added to Pivotal story (if relevant)
- [ ] Follow-up stories added to Pivotal

# Reviewer PR Checks
_(tick as appropriate)_

- [ ] PR completes task/fixes bug
- [ ] Design makes sense, and fits with our current code base
- [ ] Code is easy to follow
- [ ] PR size is sensible
- [ ] Commit history is sensible and tidy

# Info on PRs
The checks above are guidelines. They don't all have to be ticked, but they should all have been considered.

### CODE REVIEW

File: api/schemas/flow-video.json [hunk 1]
```diff
@@ -94,7 +94,7 @@
                             ]
                         },
                         "aspect_ratio": {
-                            "description": "The display aspect ratio.",
+                            "description": "The display aspect ratio. i.e. pixel_width / pixel_height",
```
Comment (philipnbbc) [inline api/schemas/flow-video.json:97]: I don't think that change is valid for SD which has the complication of active areas etc. and pixel dimensions can vary when display aspect ratio should be set to 16:9 or 4:3 for example.

File: api/schemas/flow-audio.json [hunk 2]
```diff
@@ -54,7 +54,7 @@
                   "type": "integer"
                 },
                 "mp4_oti": {
-                  "description": "The MPEG-4 Object Type Identification.",
+                  "description": "The MPEG-4 Object Type Identification. For more information on this property, see https://developer.mozilla.org/en-US/docs/Web/Media/Formats/codecs_parameter#mpeg-4_audio",
```
Comment (philipnbbc) [inline api/schemas/flow-audio.json:57]:
```suggestion
                  "description": "The MPEG-4 Object Type Identification. For more information on the use of this property in codec strings, see https://developer.mozilla.org/en-US/docs/Web/Media/Formats/codecs_parameter#mpeg-4_audio",
```

File: api/schemas/flow-video.json [hunk 3]
```diff
@@ -112,7 +112,7 @@
                             }
                         },
                         "pixel_aspect_ratio": {
-                            "description": "The pixel aspect ratio.",
+                            "description": "The pixel aspect ratio. This is usually 1:1 (i.e. square pixels) for modern video. Older Standard Definition widescreen video often used non-square pixels. This is where that may be indecated.",
```
Comment (philipnbbc) [inline api/schemas/flow-video.json:115]:
```suggestion
                            "description": "The pixel aspect ratio. This is usually 1:1 (i.e. square pixels) for modern video. Older Standard Definition widescreen video often used non-square pixels. This is where that may be indicated.",
```

File: api/schemas/flow-video.json [hunk 4]
```diff
@@ -186,15 +186,15 @@
                             ],
                             "properties": {
                                 "profile": {
-                                    "description": "AVC / H.264 profile byte.",
+                                    "description": "AVC / H.264 profile byte. For more information on this property, see https://developer.mozilla.org/en-US/docs/Web/Media/Formats/codecs_parameter#using_the_codecs_parameter",
```
Comment (philipnbbc) [inline api/schemas/flow-video.json:189]:
```suggestion
                                    "description": "AVC / H.264 profile byte. For more information on the use of this property in codec strings, see https://developer.mozilla.org/en-US/docs/Web/Media/Formats/codecs_parameter#using_the_codecs_parameter",
```

File: api/schemas/flow-video.json [hunk 5]
```diff
@@ -186,15 +186,15 @@
...
                                     "type": "integer"
                                 },
                                 "level": {
-                                    "description": "AVC / H.264 level byte.",
+                                    "description": "AVC / H.264 level byte. For more information on this property, see https://developer.mozilla.org/en-US/docs/Web/Media/Formats/codecs_parameter#using_the_codecs_parameter",
```
Comment (philipnbbc) [inline api/schemas/flow-video.json:193]:
```suggestion
                                    "description": "AVC / H.264 level byte. For more information on the use of this property in codec strings, see https://developer.mozilla.org/en-US/docs/Web/Media/Formats/codecs_parameter#using_the_codecs_parameter",
```

File: api/schemas/flow-video.json [hunk 6]
```diff
@@ -186,15 +186,15 @@
...
                                     "type": "integer"
                                 },
                                 "flags": {
-                                    "description": "AVC / H.264 flags byte.",
+                                    "description": "AVC / H.264 flags byte. For more information on this property, see https://developer.mozilla.org/en-US/docs/Web/Media/Formats/codecs_parameter#using_the_codecs_parameter",
```
Comment (philipnbbc) [inline api/schemas/flow-video.json:197]:
```suggestion
                                    "description": "AVC / H.264 flags byte. For more information on the use of this property in codec strings, see https://developer.mozilla.org/en-US/docs/Web/Media/Formats/codecs_parameter#using_the_codecs_parameter",
```

**Review by philipnbbc — APPROVED** [review]
A couple inline comments and otherwise LGTM. I thought that adding "on the use of this property in codec strings" would be better as the reference isn't for defining this property, although it does explain what the property is.

File: api/schemas/flow-video.json (hunk 1 above)
Comment (j616) [reply api/schemas/flow-video.json]: Sorted in fixup

File: api/schemas/flow-audio.json (hunk 2 above)
Comment (j616) [reply api/schemas/flow-audio.json]: Sorted in fixup

File: api/schemas/flow-video.json (hunk 3 above)
Comment (j616) [reply api/schemas/flow-video.json]: Sorted in fixup

File: api/schemas/flow-video.json (hunk 4 above)
Comment (j616) [reply api/schemas/flow-video.json]: Sorted in fixup

File: api/schemas/flow-video.json (hunk 5 above)
Comment (j616) [reply api/schemas/flow-video.json]: Sorted in fixup

File: api/schemas/flow-video.json (hunk 6 above)
Comment (j616) [reply api/schemas/flow-video.json]: Sorted in fixup

File: api/schemas/flow-video.json [hunk 7]
```diff
@@ -112,7 +112,7 @@
                             }
                         },
                         "pixel_aspect_ratio": {
-                            "description": "The pixel aspect ratio.",
+                            "description": "The pixel aspect ratio. This is usually 1:1 (i.e. square pixels) for modern video. Older Standard Definition widescreen video often used non-square pixels. This is where that may be indicated.",
```
Comment (himslm01) [inline api/schemas/flow-video.json:115]: non-widescreen was also often non-square pixels too

**Pull request merged**